tests/test_indexer.py

```diff
@@ -12,10 +14,6 @@
 import uuid
 from http.server import BaseHTTPRequestHandler, HTTPServer
 
-import google.auth
```

Comment:
Nit: I'm totally fine with sneaking in such small fixes into an unrelated PR as long as they are in a separate commit AND the PR lands unsquashed.